SabreTools.Models/SecuROM/DFAFile.cs

```diff
@@ -0,0 +1,114 @@
+namespace SabreTools.Models.SecuROM
+{
+    /// <summary>
+    /// Most DFA-protected files seem to also have some additional encryption (it looks like SecuROM DFE, but that
+    /// could be an incorrect guess); only early RC-encrypted executables seem to have anything parsable beyond the
+    /// SDFA header. When I mention "observable cases" this is what I mean.
+    /// </summary>
+    public class DFAFile
+    {
+        // Header Start
```

Comment:
If this is a separate concept then it should be a separate model that is just referenced here instead.